manifests/agent.pp

```diff
@@ -177,6 +177,9 @@
 # [*loadmodule*]
 #   Module to load at agent startup.
 #
+# [*manage_init_script*]
```

Comment:
`zabbix::startup` manages systemd instead of init scripts on some systems, so this parameter name may not be the best choice.

Reply:
works for me.  @bastelfreak - sound ok?

Reply:
sounds good to me. @baurmatt can you update it please?

Reply:
Done. Didn't change the branch name though ;)